src/components/Grants/GrantCard/ProfileGrantCard.tsx

```diff
@@ -0,0 +1,27 @@
+import React, { useMemo } from 'react'
+
+import { GrantAttributes } from '../../../entities/Proposal/types'
+import { useSortingByKey } from '../../../hooks/useSortingByKey'
+
+import ProfileGrantItem from './ProfileGrantItem'
+
+interface Props {
+  grants: GrantAttributes[]
+}
+
+const MAX_GRANTS = 4
+
+function ProfileGrantCard({ grants }: Props) {
```

Comment:
This is subtle, so I open the discussion. 
I think i'd rename this to `ProfileGrantCards`, or `ProfileGrants` or `ProfileGrantsSection`, and each `ProfileGrantItem` to `ProfileGrantCard`.

Just because on the other pages (home and grants) we have cards for grants, and there's a one to one relationship between a card and a grant. 

It seems strange for me to have a card with several grants inside. 

WDYT?

Reply:
`ProfileGrantList`?